packages/wrangler/src/index.tsx

```diff
@@ -2244,8 +2244,20 @@ export async function main(argv: string[]): Promise<void> {
     }
   );
 
-  wrangler.command("pages", "⚡️ Configure Cloudflare Pages", (pagesYargs) =>
-    pages(pagesYargs.command(subHelp))
+  wrangler.command(
+    "pages",
+    "⚡️ Configure Cloudflare Pages",
+    async (pagesYargs) => {
+      await pages(
+        pagesYargs
+          .command(subHelp)
+          .epilog(
+            chalk.yellow(
+              "'wrangler pages <command>' is a Beta command. Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose"
```

Comment:
will this message show for every pages command?

Reply:
Yes. It run anytime the command is utilized. 

If that is too often, alternatively, it could be added in the help message.

Reply:
Doesn't seem to when I'm testing?

```
npx wrangler pages --help
```

> `'wrangler pages <command>' is a Beta command. Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose`

```
npx wrangler pages dev --help
```

No warning message.

```
npx wrangler pages dev <dir>
```

No warning message.

Reply:
good catch, retracting my stamp

Reply:
I will add more comprehensive testing around it, and fix that.

Reply:
unit testing subcommands was not feasible, but the tests in examples showed it working as expected on subcommand usage. 

Screenshots added to PR